product_variant_configurator/models/product_template.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2015 Oihane Crucelaegui - AvanzOSC
+# Copyright 2016 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# Copyright 2016 ACSONE SA/NV
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3
+
+from odoo import _, api, fields, models
+from odoo.tools import config
+
+
+class ProductTemplate(models.Model):
+    _inherit = 'product.template'
+
+    no_create_variants = fields.Selection(
+        [('yes', "Don't create them automatically"),
```

Comment:
maybe it would be better to invert this field or use no_create and create for the values.